memo/ts/package.json

```
@@ -1,18 +1,26 @@
 {
   "name": "@solana/spl-memo",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "SPL Memo Program JS API",
   "files": [
-    "dist",
+    "lib",
     "src"
   ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.mjs",
+  "types": "lib/types/index.d.ts",
   "type": "module",
   "sideEffects": false,
+  "exports": {
+    "import": "./lib/esm/index.mjs",
+    "require": "./lib/cjs/index.js"
```

```suggestion
    "require": "./lib/cjs/index.js",
    "types": "./lib/types/index.d.ts"
```

We haven't done this in many other places yet, but adding this extra `types` path helps some IDEs and build tools.